src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -192,7 +192,7 @@ public void updateFilteredIncomeList(Predicate<Entry> predicate) {
      */
     public ObservableList<PieChart.Data> getIncomePieChartData() {
 
-        int[] incomePieChartArr = new int[6];
+        double[] incomePieChartArr = new double[6];
```

Comment:
Changed this to `double` array since there were some rounding issues with the test cases.

@sharmaine1028 @yuanxi1 lmk if this affects anything!